modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/MetadataTypeUtils.java

```diff
@@ -133,4 +136,8 @@ public static boolean isExtensible(MetadataType metadataType) {
   public static boolean isEnum(MetadataType metadataType) {
     return metadataType.getAnnotation(EnumAnnotation.class).isPresent();
   }
+
+  public static Set<MetadataKey> getKeysFromContainer(MetadataKeysContainer metadataKeysContainer) {
+    return metadataKeysContainer.getKeys(metadataKeysContainer.getResolvers().iterator().next()).get();
```

Comment:
That was my first approach (adding it to MetadataExtensionFunctionalTestCase) but then I needed the same method in another test from another module (in extension-support and extension-spring-support).